requirements/analyzer.txt

```diff
@@ -122,6 +123,37 @@ numpy==1.26.4
     #   thinc
     #   torchvision
     #   transformers
+nvidia-cublas-cu12==12.1.3.1
+    # via
+    #   nvidia-cudnn-cu12
+    #   nvidia-cusolver-cu12
+    #   torch
+nvidia-cuda-cupti-cu12==12.1.105
+    # via torch
+nvidia-cuda-nvrtc-cu12==12.1.105
+    # via torch
+nvidia-cuda-runtime-cu12==12.1.105
+    # via torch
+nvidia-cudnn-cu12==8.9.2.26
+    # via torch
+nvidia-cufft-cu12==11.0.2.54
+    # via torch
+nvidia-curand-cu12==10.3.2.106
+    # via torch
+nvidia-cusolver-cu12==11.4.5.107
```

Comment:
I wonder why you got a bunch of cuda dependencies that I didn't 🤔 Perhaps different in OS or something?

Reply:
I'm not sure exactly. Definitely odd.

Reply:
I did run this again while fixing up #895 and got the same things.